paper/paper.md

```diff
@@ -43,60 +43,85 @@ bibliography: paper.bib
 
-To satisfy our research need we thus opted to develop lightweight simulation models for all involved systems that would allow for around 120 thousand simulation runs together with the component sizing and life-cycle cost analysis to be performed on a computer with a 12-core processor in about 8 hours. The users can expect a single solar water heater simulation model to run for about 0.16 seconds, providing an almost instantaneous experience for a user only seeking to design and investigate a single system.
+For those who in their professional life deal with planning, designing, and contracting the solar thermal water heating systems it might be useful to have an access to a freely available simulation tool, such as the MSWH software, that they can use to evaluate various system designs. The design parameters that such users can easily modify are household occupancies, climate zone, collector and tank sizes, component performance parameters such as insulation level of any thermal storage tanks, and types of solar collectors. The MSWH software relies on standard collector rating data readily available for most designs found on the market today. For each proposed design the MSWH software will output, among other results, the solar fraction and the backup energy use on an annual level, the two variables allowing for a quick cross-comparison for the proposed designs.
+
+Similarly to the previous category, the homeowners considering transitioning to a solar water heating system may be interested in doing the math before seeking further professional help, or just for their own education and curiosity about both solar water heating systems and system simulation in general. Another example use case would be to enable the occupants of households that:
+
+* Are retrofitting an existing system due to an increase or decrease the occupancy, or
+* Already possess one of the components and are looking to appropriately size the others
+
+to simulate alternatives and compare the obtained energy consumption and solar fraction results for any alternative designs they like to define.
 
-Apart from the various research applications, the MSWH package can be interesting for homeowners, planners, designers, policy developers or even solar water heating contractors, just to list a few.
+Lastly, simulation tools tend to be inaccessible to non-technical users, both in terms of usage and the chance for the user to understand the underlying codebase just by reading through it. The MSWH software provides a unique insight into what actually happens in a relatively simple mezzo-level simulation model due to the usage of relatively well readable Python code, while the example notebooks and GUI allow for instant utilization of the models. These features make the code well suitable also for educators.
```

Comment:
"The MSWH software provides a unique insight into what actually happens in a relatively simple mezzo-level simulation model due to the use of readable Python code, while the example notebooks and GUI allow for instant utilization of the models. These features make the code suitable also for educators."